Tests/LibGfx/TestImageDecoder.cpp

```diff
@@ -2137,6 +2137,15 @@ TEST_CASE(test_jxl_modular_property_8)
     verify_checkerboard(*plugin_decoder);
 }
 
+TEST_CASE(test_jxl_modular_squeeze)
```

Comment:
can you say in the commit description how you created the test input?

Reply:
Sure, thanks for the reminder!